src/main/java/seedu/jarvis/commons/exceptions/CourseNotFoundException.java

```diff
@@ -0,0 +1,14 @@
+package seedu.jarvis.commons.exceptions;
+
+/**
+ * Signals that a course could not be found.
+ */
+public class CourseNotFoundException extends Exception {
```

Comment:
Might be useful to have a default message in a nullary constructor for convenience.